config/manager/kustomization.yaml

```diff
@@ -5,6 +5,12 @@ generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
-- name: manager-config
-  files:
+- files:
   - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/tweining/octavia-operator
```

Comment:
This gets set whenever you build your own image for use in your quay, but we'll eventually want this to have `quay.io/openstack-k8s-operators/octavia-operator` before we merge

Reply:
How does the image get pushed to `quay.io/openstack-k8s-operators/octavia-operator`? Is the CI doing that automatically? I will change this line of course.

Reply:
I didn't mean to imply that it's being pushed there.  Whenever you build/push to anywhere, this line gets set to wherever you pushed.  So it's just a manual process to change it to be `quay.io/openstack-k8s-operators/octavia-operator` (since you are _not_ pushing there yourself) before the PR is finished.

Reply:
> How does the image get pushed to `quay.io/openstack-k8s-operators/octavia-operator`? Is the CI doing that automatically? I will change this line of course.

just a small follow up on this question. in .github/workflows there is one to build and push the new operator when merged.